TestProject/src/rules/BaseRule.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace GameOfLife\Rules;
+
+use GameOfLife\options\Getopt;
+use GameOfLife\cellularAutomat\Field;
+
+/**
+ * API-class for additional rules.
+ */
+abstract class BaseRule
+{
+    /**
+     * @param Field $_field The cell which is going to be checked.
+     * @return bool The new state of the given field in the next generation
+     */
+    public function calculateNewState(Field $_field)
+    {
+    }
```

Comment:
Since there is no default implementation for this, you should define the class as abstract and remove the { and }.